hugegraph-api/src/main/java/com/baidu/hugegraph/api/graph/BatchAPI.java

```diff
@@ -125,7 +125,7 @@ protected void updateExistElement(HugeGraph g,
             String key = kv.getKey();
             UpdateStrategy updateStrategy = kv.getValue();
             if (oldElement.property(key).isPresent() &&
-                newElement.properties.get(key) != null ) {
```

Comment:
remember to modify these:
1. hugegraph-api pom.xml `<Implementation-Version>0.39.0.0</Implementation-Version>` to `<Implementation-Version>0.40.0.0</Implementation-Version>`
2. hugegraph-api ApiVersion.class `0.39` to `0.40`, and add an item under `[0.39] Issue-522: Add profile RESTful API`, write something like `[0.40] Issue-493: Support batch updating elements' property by multiple strategy`